src/main.ts

```diff
@@ -43,6 +40,10 @@ const router = createRouter({
       path: '/:lang(en-us|es-es)',
       component: Home,
     },
+    {
+      path: '/?infoMessage=:infoMessage(.*)',
```

Comment:
You added a new route?

Reply:
yes, when the infomessage is shown 
![image](https://user-images.githubusercontent.com/54802833/233933927-e175ffc4-1aec-4ab6-a6e6-8093d0701d32.png)